bundles/org.dataflowanalysis.analysis.converter/src/org/dataflowanalysis/analysis/converter/microsecend/MicroSecEndProcess.java

```diff
@@ -0,0 +1,51 @@
+package org.dataflowanalysis.analysis.converter.microsecend;
+
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+public class MicroSecEndProcess {
+    @JsonProperty("name")
+    protected String name;
+    @JsonProperty("stereotypes")
+    protected List<String> stereotypes;
+    @JsonProperty("tagged_values")
+    @JsonDeserialize(using = TaggedValuesDeserializer.class)
+    protected Map<String, List<String>> taggedValues;
+
+    public MicroSecEndProcess() {
+
+    }
```

Comment:
Is this intended? This creates a class with all null attributes with no way to change them

Reply:
Forgot to make this abstract